gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -687,7 +687,6 @@ private void cancelDagNode(DagNode<JobExecutionPlan> dagNodeToCancel) throws Exe
         Future future = dagNodeToCancel.getValue().getJobFuture().get();
         String serializedFuture = DagManagerUtils.getSpecProducer(dagNodeToCancel).serializeAddSpecResponse(future);
         props.put(ConfigurationKeys.SPEC_PRODUCER_SERIALIZED_FUTURE, serializedFuture);
-        sendCancellationEvent(dagNodeToCancel.getValue());
```

Comment:
Have concerns on this line. After removing this, are we able to handle normal cancel requests?

Reply:
crap good catch, I didn't realize it was also used for manual kill requests as well. Will add a parameter to not send the event on SLA kill and let the pollAndAdvanceDag() make the decision there